app/models/user.rb

```diff
@@ -463,4 +464,11 @@ def strip_whitespace_from_name
   def update_password_changed
     self.password_changed_at = Time.current if (new_record? || encrypted_password_changed?) && !password_changed_at_changed?
   end
+
+  def record_update
```

Comment:
I think at this point I was relying on the unit test(s) in `UserUpdateTest`. However, it might well make sense to move those to `UserTest` in this commit (or possibly a subsequent commit).